### PR TITLE
ui: replace momentjs with dayjs

### PR DESCRIPTION
**Pull request by Rustin-Liu**
closes https://github.com/pingcap-incubator/tidb-dashboard/issues/181

### CODE REVIEW

File: ui/lib/components/DatePicker/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Dayjs } from 'dayjs'
```
Comment (breeswish) [inline ui/lib/components/DatePicker/index.tsx:1]: I think you also need to expose it in `components/index.ts`.

File: ui/lib/apps/SearchLogs/components/SearchResult.tsx [hunk 2]
```diff
@@ -11,6 +10,7 @@ import { LogLevelText } from '../utils'
 import Log from './Log'
 
 import styles from './Styles.module.less'
+import dayjs from 'dayjs'
```
Comment (breeswish) [inline ui/lib/apps/SearchLogs/components/SearchResult.tsx:13]: prefer to put it before L9

File: ui/lib/components/DatePicker/index.tsx (hunk 1 above)
Comment (Rustin-Liu) [reply ui/lib/components/DatePicker/index.tsx:1]: Added.

File: ui/lib/apps/SearchLogs/components/SearchResult.tsx (hunk 2 above)
Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchResult.tsx:13]: Fixed.

**Pull request merged**